ydb/docs/en/core/integrations/ingestion/spark.md

```diff
@@ -0,0 +1,298 @@
+# {{ spark-name }}
+
+{{ spark-name }} is a fast, open-source cluster computing system for big data processing that works with various data stores and supports multiple programming languages (Scala, Java, Python, R). {{ spark-name }} can work with {{ ydb-full-name }} using [Spark Connector](https://github.com/ydb-platform/ydb-spark-connector), a special module that implements core  {{ spark-name }} primitives.
```

Comment:
This article goes beyond just `ingestion/` as it covers reading data back as well. Let's at least acknowledge this in the intro paragraph, as we're avoiding creating a new integration category for now.